lib/rules/declaration-property-value-keyword-no-deprecated/index.mjs

```diff
@@ -129,10 +129,17 @@ const rule = (primary, secondaryOptions) => {
 			if (!keywords) return;
 
 			const parsedValue = valueParser(getDeclarationValue(decl));
+			let flag = false;
 
 			parsedValue.walk((node) => {
+				if (flag) return;
+
 				if (node.type !== 'word') return;
 
+				// FIXME once we support shorthands
+				// e.g. border-color: green red yellow ActiveBorder;
+				flag = true;
```

Comment:
@ybiquitous what's better in your opinion a flag like I did or `try`/`catch`?

Reply:
A `throw` + `try/catch` is extremely slow.
When an actual exception needs to happen or needs to caught, then this is fine. But using it as a way to escape early will most likely be slower than just letting the `walk` run to completion.

Reply:
> A `throw` + `try/catch` is extremely slow.

I know…